hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/key/OMKeyCommitRequestWithFSO.java

```diff
@@ -193,7 +193,7 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
       }
 
       // Add to cache of open key table and key table.
-      OMFileRequest.addOpenFileTableCacheEntry(omMetadataManager, dbFileKey,
+      OMFileRequest.addOpenFileTableCacheEntry(omMetadataManager, dbOpenFileKey,
```

Comment:
Nice catch.

So, what happens is when an FSO key is committed, the open-key entry is not cleaned up. Logically, that should prevent the key from being reopened, e.g. for a overwrite. 
Fortunately (or maybe not), when OM opens a key for writing, it just doesn't care if the file has been opened already (even by the same client). I feel like this behavior results in some funny scenarios when multiple clients open and commit the same file. Guess it's something we can think about.

Reply:
@duongkame did [this](https://github.com/apache/ozone/pull/4156#pullrequestreview-1242797126) answer your question or do you still have some concerns about this change?